module/importer/src/Infrastructure/Action/SimpleProductImportAction.php

```diff
@@ -28,15 +31,25 @@ public function action(
         Sku $sku,
         string $template,
         array $categories,
-        array $attributes = []
+        array $attributes = [],
+        ImportId $importId = null
     ): SimpleProduct {
         $templateId = $this->templateQuery->findTemplateIdByCode($template);
         if (null === $templateId) {
             throw new ImportException('Missing {template} template.', ['{template}' => $template]);
         }
         $productId = $this->productQuery->findProductIdBySku($sku);
         $categories = $this->getCategories($categories);
-        $attributes = $this->builder->build($attributes);
+
+        try {
+            $attributesBuilt = $this->builder->build($attributes);
```

Comment:
Won't this block other attributes? And eventually, we may end up with not fully imported attributes due to this specific one?